### ISSUE TITLE
on-page anchor links render title beneath (hidden by) header

### ISSUE THREAD

**Issue by chrisconnors-ibm** [carbon-design-system/carbon-website#1464]
## Detailed description

if you navigate to https://www.carbondesignsystem.com/guidelines/accessibility/overview#carbon-and-accessibility by clicking it from an external source (like slack) the header anchored to `#carbon-and-accessibility` is properly spaced from the header:
![image](https://user-images.githubusercontent.com/35537391/55640143-0363dd80-5799-11e9-8b2d-3ad6ba82ce1f.png)

If you click the anchor link icon 
![image](https://user-images.githubusercontent.com/35537391/55640270-46be4c00-5799-11e9-85cd-c8046a4573e1.png)
on the page next to the header, that same h2 anchored is scrolled up and hidden by the header:
![image](https://user-images.githubusercontent.com/35537391/55640318-589fef00-5799-11e9-9c88-89b6cf0750f3.png)

Those two actions should have the same result: the header anchored to `#carbon-and-accessibility` is properly spaced from the header:
![image](https://user-images.githubusercontent.com/35537391/55640143-0363dd80-5799-11e9-8b2d-3ad6ba82ce1f.png)
